### PR TITLE
add the possibility to trace the time in log

### PR DESCRIPTION
**Pull request by antoine-de**
you can now trace the timestamp in log with the env var `LOG_TIME`

### CODE REVIEW

**Comment by TeXitoi** [comment]
Why not just doing that by default?

**Comment by antoine-de** [comment]
it adds verbosity to the log, but I'm ok with including this by default

**Comment by antoine-de** [comment]
done

File: libs/mimir/src/lib.rs [hunk 1]
```diff
@@ -48,12 +48,20 @@ extern crate geojson;
...
-    builder.filter(None, log::LogLevelFilter::Info);
+
+    builder.filter(None, log::LogLevelFilter::Info)
+        .format(|record| {
+            format!("[{time}]{lvl}:{loc}: {msg}",
+                    time = Local::now(),
+                    lvl = record.level(),
+                    loc = record.location().module_path(),
+                    msg = record.args())
+        });;
```
Comment (TeXitoi) [inline libs/mimir/src/lib.rs]: double `;`?

Comment (antoine-de) [inline libs/mimir/src/lib.rs]: oups mispaste :confused:

**Pull request merged**